src/components/NcSelect/NcSelect.vue

```diff
@@ -491,6 +461,13 @@ export default {
 		v-bind="propsToForward"
 		v-on="$listeners"
 		@search="searchString => search = searchString">
+		<template #header>
+			<label v-if="inputLabel"
+				:for="inputId"
```

Comment:
For this, because inputLabel has a default value, this will never evaluate to false, so you can remove the v-if

Reply:
This is needed when a falsy inputLabel value is passed explicitly, unlikely but would handle that use case